parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -307,8 +307,8 @@ class ModelChatBlueprintArgs(BaseModelChatBlueprintArgs):
     conversation_start_mode: str = field(
         default='hi',
         metadata={
-            "help": 'Whether to show "Hi!" or two previous utterances (as in BlendedSkillTalk) at the beginning of the conversation',
-            "choices": ['hi', 'bst'],
+            "help": 'Whether to show "Hi!" or two previous utterances (as in BlendedSkillTalk) at the beginning of the conversation '
+            'or specified by specific task name'
```

Comment:
nit: maybe this would be clearer as `'Set to "hi" to show "Hi!" at the beginning of the conversation, or set to a task name to specify a custom context'`